src/TokenHandler/AccountManager.mo

```diff
@@ -173,7 +194,6 @@ module {
 
       switch (transferResult) {
```

Comment:
Should we have two-layer `transferResult` handling or we can simply return `transferResult` from `processConsolidationTransfer()` and handle the response completely in `consolidate()`?

Reply:
Pushed this improvement, review please